engine/table/src/main/java/io/deephaven/engine/util/GroovyDeephavenSession.java

```diff
@@ -62,18 +90,25 @@ public class GroovyDeephavenSession extends AbstractScriptSession<GroovySnapshot
     private static final String DEFAULT_SCRIPT_PATH = Configuration.getInstance()
             .getStringWithDefault("GroovyDeephavenSession.defaultScriptPath", ".");
 
+    private static final boolean INCLUDE_DEFAULT_IMPORTS_IN_LOADED_GROOVY =
+            Configuration.getInstance()
+                    .getBooleanWithDefault("GroovyDeephavenSession.includeDefaultImportsInGroovyScripts", false);
+    private static final boolean INCLUDE_CONSOLE_IMPORTS_IN_LOADED_GROOVY =
```

Comment:
I view INCLUDE_CONSOLE_IMPORTS_IN_LOADED_GROOVY as having close to no (clear) value. To have effect, there would need to be scripts that rely on (or at least can be affected by) imports from the console, which is obviously not useful for prod, but also not useful for developing prod queries.

Maybe it can help some unlikely and/or fringe cases, but we might even purposefully not want to support those to advocate against such tight coupling in our system.

I'm requesting the removal of this prop on initial release. And if it does end up being wanted by a user later on, we can consider adding then.

Reply:
@rbasralian any objection? this will be easy to revert if we decide we need it. When gathering requirements from Arman to begin with, I had mistakenly added this as we reviewed what other features might matter, and I think I agree that the utility here is very limited.

Reply:
I'm ok with that.

The benefit of this is that it lets you write scripts scripts (e.g. different notebook) without having to worry about copying the same imports to every single one, which is just a hassle to do, especially in the web UI where you can't open local scripts like you could in Swing (so you could just do all editing in IntelliJ and let it worry about the imports). But I agree that it's not the best practice, and now that we load Groovy scripts properly we should push users toward handling imports properly.